src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IConstructorBodyOperation.cs

```diff
@@ -150,8 +196,39 @@ public C() : base()
           IThrowOperation (OperationKind.Throw, Type: null) (Syntax: 'throw null')
             ILiteralOperation (OperationKind.Literal, Type: null, Constant: null) (Syntax: 'null')
 ");
+
+            VerifyFlowGraph(compilation, node1, expectedFlowGraph:
+@"
+Block[B0] - Entry
+    Statements (0)
+    Next (Regular) Block[B1]
+Block[B1] - Block
+    Predecessors: [B0]
+    Statements (1)
+        IExpressionStatementOperation (OperationKind.ExpressionStatement, Type: null, IsImplicit) (Syntax: ': base()')
+          Expression: 
+            IInvocationOperation ( System.Object..ctor()) (OperationKind.Invocation, Type: System.Void) (Syntax: ': base()')
+              Instance Receiver: 
+                IInstanceReferenceOperation (ReferenceKind: ContainingTypeInstance) (OperationKind.InstanceReference, Type: System.Object, IsImplicit) (Syntax: ': base()')
+              Arguments(0)
+
+    Next (Throw) Block[null]
+        ILiteralOperation (OperationKind.Literal, Type: null, Constant: null) (Syntax: 'null')
+Block[B2] - Block [UnReachable]
+    Predecessors (0)
+    Statements (1)
+        IExpressionStatementOperation (OperationKind.ExpressionStatement, Type: null, IsImplicit) (Syntax: 'throw null')
```

Comment:
>IExpressionStatementOperation (OperationKind.ExpressionStatement, Type: null, IsImplicit) (Syntax: 'throw null') [](start = 8, length = 112)

Where does this come from?

Reply:
I see what is going on. It looks like we need to adjust implementation of VisitExpressionStatement to return null if its child is a Throw operation, after visiting the child of course. Could you please make this small adjustment, since this affects baselines that you are adding?

---
In reply to: [189704520](https://github.com/dotnet/roslyn/pull/26955#discussion_r189704520) [](ancestors = 189704520)

Reply:
I was going to come talk to you about the difference between expression bodied and block case, thanks for suggesting the correct approach.

---
In reply to: [189708626](https://github.com/dotnet/roslyn/pull/26955#discussion_r189708626) [](ancestors = 189708626,189704520)